cns/service/main.go

```diff
@@ -168,7 +176,7 @@ func main() {
 	ipamQueryInterval, _ := acn.GetArg(acn.OptIpamQueryInterval).(int)
 	stopcnm = acn.GetArg(acn.OptStopAzureVnet).(bool)
 	vers := acn.GetArg(acn.OptVersion).(bool)
-	// reportToHostInterval := acn.GetArg(acn.OptReportToHostInterval).(int)
+	reportToHostInterval := acn.GetArg(acn.OptReportToHostInterval).(int)
```

Comment:
why do you need this?  telemetry service will read this from config. client code should not have it

Reply:
also remove OptReportToHostInterval where its decalred

Reply:
removed